packages/fxa-admin-panel/package.json

```diff
@@ -84,7 +84,7 @@
     "prettier": "^2.3.1",
     "supertest": "^6.3.0",
     "tailwindcss-textshadow": "^2.1.3",
-    "typescript": "^5.3.3",
+    "typescript": "^5.4.2",
```

Comment:
Can this and the other package-based references be removed?

Reply:
Unfortunately at the moment no. See this comment from PR #16390.

> Readd a few dev dependancies back into packages/**/package.json
>
>  Behavior of npx nx... commands seems to have changed, resulting in
>  multiple "command not found: " errors, where
>  refers to libaries used in packages/**/package.json scripts, such as
>  tailwindcss, tsc-alias, nx, etc.